Dependencies/xr/Include/XR.h

```diff
@@ -169,6 +190,7 @@ namespace xr
             void RequestEndSession();
             Size GetWidthAndHeightForViewIndex(size_t viewIndex) const;
             void SetDepthsNearFar(float depthNear, float depthFar);
+            void SetPlaneDetectionEnabled(bool enabled)const;
```

Comment:
This, I think, makes more sense on the `Session` than here since it's not really a setting of the frame, but of the session.

Reply:
This is already on the session.

Reply:
Ha! Ignore me. 😆